clocks/meetings/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+
+class Meeting(models.Model):
+    room_id = models.ForeignKey(
```

Comment:
```suggestion
    room = models.ForeignKey(
```
это же уже по сути объект